### PR TITLE
MR | Simple and badge profile list updated

### PR DESCRIPTION
**Pull request by moshfiqrony**
## What type of PR is this? (check all applicable)

- [x] 🚀 Added Name
- [ ] ✨ Feature
- [ ] ✅ Joined Community
- [x] 🌟 ed the repo
- [ ] 🐛 Grammatical Error
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Description
I have updated two entries for Badge and simple profiles. I have added some new badges like - Express, graphql, Jenkins, VS code, shell, and many more

## Add Link of GitHub Profile
https://github.com/moshfiqrony

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@
 - [Lucas Vazquez](https://github.com/lucasvazq/lucasvazq)
 - [小弟调调™](https://github.com/jaywcjlove/jaywcjlove)
 - [Johnny Villegas](https://github.com/C9-LinkRs/C9-LinkRs)
+- [Moshfiq Rony](https://github.com/moshfiqrony)
```
Comment (abhisheknaiidu) [inline README.md:99]: Remove this, just add it under `badges`

**Comment by moshfiqrony** [comment]
I have removed and kept the only badge entry

File: README.md [hunk 2]
```diff
@@ -143,6 +143,7 @@
 - [Anirudh Emmadi](https://github.com/aemmadi/aemmadi)
 - [Brendon Smith](https://github.com/br3ndonland/br3ndonland)
 - [Alwin Wang](https://github.com/alwinw/alwinw)
+- [Moshfiq Rony](https://github.com/moshfiqrony)
```
Comment (abhisheknaiidu) [inline README.md:146]: change it from `https://github.com/moshfiqrony` to `https://github.com/moshfiqrony/moshfiqrony`

**Comment by moshfiqrony** [comment]
I have done all the changes

**Pull request merged**